bokehjs/src/lib/core/has_props.ts

```diff
@@ -260,6 +260,16 @@ export abstract class HasProps extends Signalable() implements Equatable, Printa
     return attrs
   }
 
+  get dirty_attributes(): Attrs {
```

Comment:
I'm not entirely sure what dirty means. Maybe add a docstring somewhere.

Reply:
Dirtiness is a well defined concept in bokehjs' properties system, though not documented, so I suppose it won't hurt to do it here.